cluster/manifests/flannel/daemonset.yaml

```diff
@@ -48,6 +48,9 @@ spec:
           limits:
             cpu: 25m
             memory: 50Mi
+        volumeMounts:
+        - name: run
+          mountPath: /run
```

Comment:
The local container filesystem isn't reachable by the `kube-flannel` container when it comes up and it ends up failing the readiness probe. I wanted a shared volume between both these containers to be able to provide the readiness probe to the other container, that already has this mounted.

Reply:
Oh, you mean the probe is needed for this container? I thought we needed the probe for `kube-flannel` container 👀

Reply:
> Oh, you mean the probe is needed for this container?

yeah, I didn't realize it was on the other one. We just need it for this one because if any of the containers fail the probe it's enough to mark the pod not ready and fail the node from getting ready to run user pods.

Reply:
Oh, that simplifies a lot of things. Great. Thanks for catching this 🙂

Reply:
Fixed in the next commit.